server/plugin.go

```diff
@@ -169,6 +172,41 @@ func (p *Plugin) syncSince(syncSince time.Time) {
 	p.API.LogDebug("Syncing since", "date", syncSince)
 }
 
+func (p *Plugin) getBase64Certificate() string {
+	certificate := p.getConfiguration().CertificatePublic
+	if certificate == "" {
+		return ""
+	}
+	block, _ := pem.Decode([]byte(certificate))
+	return base64.StdEncoding.EncodeToString(pem.EncodeToMemory(block))
+}
+
+func (p *Plugin) getPrivateKey() (*rsa.PrivateKey, error) {
+	keyPemString := p.getConfiguration().CertificateKey
+	if keyPemString == "" {
+		return nil, errors.New("certificate private key not configured")
+	}
+	privPem, _ := pem.Decode([]byte(keyPemString))
+	var err error
+	var parsedKey interface{}
+	if parsedKey, err = x509.ParsePKCS1PrivateKey(privPem.Bytes); err != nil {
```

Comment:
Why are we checking all these 3 paddings?

Reply:
To be honest, I was using an example of handling padding from certain go blog post, so... that was the approach to ensure everything was working there. But the padding should be an specific one, so I should change it to the one that is defined in the spec. I'll do the change.

Reply:
Thanks!